### ISSUE TITLE
[BUG]: Server responds with too many requests

### ISSUE THREAD

**Issue by likeadragonmaid** [azukaar/Cosmos-Server#216]
### What happened?

When using nextcloud and doing new backups nextcloud client(s) makes too many requests in short amount of time resulting in server responding with "Too many requests" error for every application hosted via cosmos.

### What should have happened?

Provide an option to disable this behaviour.

### How to reproduce the bug?

Use nextcloud to upload large amount of files in a short amount of time.

### System details

- OS: Fedora Server 39
- Browser: Chrome, Firefox
- Version: 0.14.6

**Comment by azukaar** [comment]
That is expected, simply instruct the smart shield to be more lenient on detection.
You can manually change the shield settings on the Nextcloud URL (make sure you dont select the DAV routes) and then in the security tab

**Comment by tarbib** [comment]
@azukaar Hello !
I ran into the same issue.
I had to disable the smart shield on the URL (servapp.domain.com) though :( changing settings to lenient did not work.
The remaining issue is I'm still locked out on domain.com where cosmos UI is accessible. I don't see how to remove smartshield on cosmos URL.
Thanks !

**Comment by azukaar** [comment]
If Lenient is not enough You can change the "Per User Request Limit " to something higher, default is 6000, try 20000 for example

**Comment by Wiz0r** [comment]
Thank you for your quick response.

The Problem was, that I could not enter the cosmos UI to make changes in the "URL" settings because of the "Too many requests" message. I disabled shield in every app via cosmos config file and restart cosmos. Then I was able to enter the cosmos UI again.

I changed "Per User Request Limit" for nextcloud to a much higher limit hoping this will prevent this massage in future.

Currently i am a bit unsure why the an app setting prevented me from logging into the cosmos UI.
